tests/utils/command_line_params.cpp

```diff
@@ -272,6 +272,12 @@ double eofa_mq3 = 1.0;
 
 QudaContractType contract_type = QUDA_CONTRACT_TYPE_OPEN;
 
+int n_steps = 50;
```

Comment:
Can you put all these variables in a struct, so that it's clear what option group they belong to?

Reply:
Thanks @maddyscientist  for the review. Not sure I understand the last point. Would you like me to create a common structure that will include all related variables? Currently I just left a comment above the group while all variables in the group remain plain build-in types. (Similar to other test vars, e.g. deflation-related ones)

Reply:
How about just prefixing these variables, e.g., `smear_n_steps`, `smeat_t0`, etc. so it's clear which group they belong to?

Reply:
sure, done.

Reply:
[35d2002](https://github.com/lattice/quda/pull/1312/commits/35d2002fcc6ce30ed86e8c59e8cd517365815db1)